src/styles.css

```diff
@@ -132,7 +135,8 @@ h2 {
 
 /* ~~~~~~~~~~ RECIPE CARD SECTION ~~~~~~~~~~~~ */
 .recipe-card {
-  border: 2px solid #1B3231;
+  /* border: 2px solid #1B3231; */
```

Comment:
we can probable delete this commented out line! unless you are saving it for something/still deciding, in that case never mind